### ISSUE TITLE
 [Adaptive Cards> ShowCardWizard.json]: Validation is not set for the fields present under 'Address and Phone/Email'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8980]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to 'ShowCardWizard.json' button by using arrow key and activate it.
4. ShowCardWizard.json adaptive card will appear on right sight of the pane.
5. Navigate to ShowCardWizard.json card by using tab key.
6. Press tab key and navigate to 'Name' button and press enter key to activate it.
7. Fill the all required fields and press the tab key and navigate to the Address button and double tap to activate it.
8. Now navigate to the 'City, state and Zip Code' and enter incorrect value.
9. Press the tab key and navigate to the 'Cell Number, Home Number and Email address' fields and enter incorrect value.
10. Press the tab key and navigate to the Submit and button and press enter key to activate it and observe the issue.

**Actual Result:**
Validation is not set for every fields present under 'Address and Phone/Email' as after activating the 'Submit button' no error message is getting displayed.

**Expected Result:**
Validation should be defined for every fields present under 'Address and Phone/Email' as after activating the 'Submit button' no error message is getting displayed. 

**User Impact:**
It will impact the user as they will block to input the valid values due to this there might be chance to enter the incorrect values.

### Screenshots

![Validation is not set for the fields present under 'Address and Phone Email'](https://github.com/user-attachments/assets/6044d614-bb88-4d97-b565-2f008e744007)

### Card JSON

https://github.com/user-attachments/assets/ec7a26ad-bd5a-4081-aa3e-8f642f0d4619

```JSON
NA
```
